documentation/updates.md

```diff
@@ -7,34 +7,48 @@ redirect_from:
 ---
 # Updates
 
-## Step 1
+## Step 1 - **Create a full backup!** 
 
-**Create a full backup!** 
+You need to backup both: your Kimai **database** and the **folder containing your Kimai installation**, especially the file `includes/autoconf.php`. To backup the folder holding your Kimai installation you can rename it and create a new folder for the new version. For thes instructions we will use the folders `public_html` and `public_html-backup`. If your webhosting use a different name adjust the instructions below as needed.
```

Comment:
Typo: For thes**e** instructions
Typo 2: your webhosting use**s** a different

Reply:
Commited